core/eval.go

```diff
@@ -87,6 +89,29 @@ func evalSET(args []string) []byte {
 				return Encode(errors.New("ERR value is not an integer or out of range"), false)
 			}
 			exDurationMs = exDurationSec * 1000
+
+		case "PXAT", "pxat":
+			i++
+			if i == len(args) {
+				return Encode(errors.New("ERR syntax error"), false)
+			}
+
+			exDurationUnixMs, err := strconv.ParseInt(args[i], 10, 64)
+			if err != nil {
+				return Encode(errors.New("ERR value is not an integer or out of range"), false)
+			}
+
+			if exDurationUnixMs < 0 {
+				return Encode(errors.New("ERR invalid expire time in 'set' command"), false)
+			}
+
+			exDurationMs = exDurationUnixMs - time.Now().UnixMilli()
```

Comment:
nit: Instead of creating a new object with expiry time set to zero, should we delete the key instead? How is it handled in Redis?